src/worker/main.ts

```diff
@@ -127,11 +128,51 @@ const sentryMiddleware = (jobHandler) => async (job) => {
 	}
 };
 
+const sendQueueMetrics = async () => {
+	if (await booleanFlag(BooleanFlags.EXPOSE_QUEUE_METRICS, false)) {
+		for (const queue of Object.values(queues)) {
+
+			const counts = {
+				active: undefined,
+				delayed: undefined,
+				failed: undefined,
+				completed: undefined,
+				paused: undefined,
+				repeatable: undefined,
+				waiting: undefined
+			};
+
+			await (Promise.all([
+				queue.getActiveCount().then((count: number) => counts.active = count),
+				queue.getDelayedCount().then((count: number) => counts.delayed = count),
+				queue.getFailedCount().then((count: number) => counts.failed = count),
+				queue.getCompletedCount().then((count: number) => counts.completed = count),
+				queue.getPausedCount().then((count: number) => counts.paused = count),
+				queue.getRepeatableCount().then((count: number) => counts.repeatable = count),
+				queue.getWaitingCount().then((count: number) => counts.waiting = count),
+			]))
```

Comment:
I would have liked to use Bull's function `getJobCountByTypes()` to get all counts at once, but it returns a single value 0 in my tests for whatever reason ...

Reply:
Couldn't you just do `getJobCounts()` instead?